### PR TITLE
Edits to Scheduled Pipelines doc

### PR DESCRIPTION
**Pull request by kbatuigas**
# Description
Some minor / additional clarifications.

# Reasons
Suggestions from review on previous PR: https://github.com/circleci/circleci-docs/pull/5939

### CODE REVIEW

**Comment by kbatuigas** [comment]
@rosieyohannan I've requested you as a reviewer so you can take a look at what's changed since the first publish!

**Review by devinmarieb — APPROVED** [review]
Thanks! @rosieyohannan has the final approval before merging :)

**Comment by rosieyohannan** [comment]
Thanks @kbatuigas I'll have a look now :-D

File: jekyll/_cci2/scheduled-pipelines.md [hunk 1]
```diff
@@ -27,10 +27,10 @@ Since the scheduled run is based on pipelines, scheduled pipelines have all the
 
 - Control the actor associated with the pipeline, which can enable the use of [restricted contexts]({{site.baseurl}}/2.0/contexts/#restricting-a-context).
 - Use [dynamic config]({{site.baseurl}}/2.0/dynamic-config/) via setup workflows.
-- Modify the schedule without having to edit `config.yml`.
-- Interact with auto-cancelling of pipelines.
-- Specify pipeline parameters associated with a schedule.
-- Consolidate the management of common schedules.
+- Modify the schedule without having to edit `.circleci/config.yml`.
+- Take advantage of auto-cancelling.
```
Comment (rosieyohannan) [inline jekyll/_cci2/scheduled-pipelines.md:31]:
```suggestion
- Take advantage of [auto-cancelling]({{site.baseurl}}/2.0/skip-build/#auto-cancelling).
```

File: jekyll/_cci2/scheduled-pipelines.md [hunk 2]
```diff
@@ -27,10 +27,10 @@ Since the scheduled run is based on pipelines, scheduled pipelines have all the
...
+- Specify [pipeline parameters]({{site.baseurl}}/2.0/pipeline-variables/#pipeline-parameters-in-configuration) associated with a schedule.
+- Manage common schedules, e.g. across workflows.
```
Comment (rosieyohannan) [inline jekyll/_cci2/scheduled-pipelines.md:33]: Sounds like a good use case for an example for the config cookbook maybe. I'll make a ticket :-)

**Review by rosieyohannan — COMMENTED** [review]
Thanks so much for these clarifications! Just one interlink addition and then good to go 🎉

File: jekyll/_cci2/scheduled-pipelines.md (hunk 2 above)
Comment (Plataformaalgoritmica) [reply jekyll/_cci2/scheduled-pipelines.md:33]: Great.

Comment (Plataformaalgoritmica) [reply jekyll/_cci2/scheduled-pipelines.md:33]: Hola Rosie, Gracias.

**Pull request merged**